react/more-react-concepts/reducing-state/reducing-state.md

```diff
@@ -0,0 +1,94 @@
+### Introduction
+
+You have probably heard about reducers before. In this lesson we are going to
+learn _what_ they are, _when_ to use them and _how_ to use them in React with the `useReducer` hook.
+
+### Lesson Overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+* What are reducers.
+* When to use reducers.
```

Comment:
The lesson doesn't really cover on `When to use reducers` or maybe I'm misreading somewhere?

Reply:
I'll expand on this, thank you.